src/components/common/Button/Toast.jsx

```diff
@@ -0,0 +1,9 @@
+import styles from './Toast.module.css';
+
+export default function Toast(props) {
+  return (
+    <div className={styles.toast}>
+      <p className={styles.message}>{props.pop}</p>
```

Comment:
```suggestion
      <p className={styles.message}>{pop}</p>
```
위의 리뷰를 이어서 적용하면 이렇게 되겠죠?